.github/workflows/ai-revision.yaml

```diff
@@ -53,6 +53,7 @@ jobs:
           AI_EDITOR_FILENAMES_TO_REVISE: ${{ inputs.file_names }}
           # specific for this manuscript https://github.com/manubot/manubot-ai-editor/issues/13
           AI_EDITOR_FILENAME_SECTION_MAPPING: '{"05.main-text.md": "methods"}'
+          AI_EDITOR_CUSTOM_PROMPT="Proofread and revise the following paragraph"
```

Comment:
note fixup in https://github.com/greenelab/connectivity-search-manuscript/commit/23a484330d76bb12d3d7d9ddcf4e3faa9515bca1